code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1188,6 +1249,40 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/alprazolam
+	name = "Alprazolam"
+	description = "Used to supress extreme cases of mental disorders."
+	reagent_state = LIQUID
+	color = "#4a07e7"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	overdose_threshold = 15
+
+/datum/reagent/medicine/alprazolam/on_mob_metabolize(mob/living/L)
+	..()
+	ADD_TRAIT(L, TRAIT_RELAXED, PSYCH_TRAIT)
+
+/datum/reagent/medicine/alprazolam/on_mob_end_metabolize(mob/living/L)
+	REMOVE_TRAIT(L, TRAIT_RELAXED, PSYCH_TRAIT)
+	..()
+
+/datum/reagent/medicine/alprazolam/on_mob_life(mob/living/carbon/M)
+	M.jitteriness = max(0, M.jitteriness-6)
+	SEND_SIGNAL(M,COMSIG_SET_SANITY,2,SANITY_NEUTRAL)
+	M.remove_status_effect(STATUS_EFFECT_SPASMS)
+	..()
+	. = 1
```

Comment:
Don't copy code without understanding why it's doing it. This is better as `return TRUE` at the end